tracing/span.go

```diff
@@ -443,8 +457,109 @@ type Headers struct {
 	Sampled *bool
 }
 
+// AnySet returns true if any of the values in the Headers are set, false otherwise.
+func (h Headers) AnySet() bool {
+	return h.TraceID != "" ||
+		h.SpanID != "" ||
+		h.Flags != "" ||
+		h.Sampled != nil
+}
+
+// TraceIDUint attempts to convert h.TraceID into a uint64, if it succeeds it
+// returns the value and 'true'.  If it fails, either because h.TraceID is not
+// set or it is malformed, ok will be 'false' and you should not rely on the ID
+// returned.
+//
+// If h.TraceID was malformed, an error will be logged using the global tracer's
+// logger but no error will be returned.
+func (h Headers) TraceIDUint() (id uint64, ok bool) {
```

Comment:
nit: Adding the type suffix to the function name feels unnecessary. I guess you want to avoid naming conflicting from the exported fields, in that case those functions can be called `Get*` instead?

Reply:
Fair, maybe `Parse*` instead?

Reply:
yea that works